modules/binary-authorization/variables.tf

```
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable attestor-name {
```

Did some digging and turns our the reason the docs were not populating within the hook is because the variable names are not quoted. Same applies for the output names as well.
```suggestion
variable "attestor-name" {
```
I couldn't find [any docs ](https://www.terraform.io/docs/configuration/variables.html)about naming variables without the double quotes although TF validate seems to work fine.

---

the 0.11 to 0.12 HCL changes kinda feel half-baked and not well documented in my opinion. I haven't done the "commit suggestion" before in GitHub, so I hope this is what I was supposed to do

---

>  the variable names are not quoted. Same applies for the output names as well.

@mike-ensor we would need to do this for all the variables as well as the outputs; the commit suggestion is neat but it only did it for one of the variables 😅

---

ah...I'll make the changes to all the files manually